docs/running_rune_with_occlum.md

```diff
@@ -54,7 +61,7 @@ RUN mkdir -p /run/rune/${OCCLUM_INSTANCE_DIR}
 WORKDIR /run/rune
 
 COPY ${OCCLUM_INSTANCE_DIR} ${OCCLUM_INSTANCE_DIR}
-COPY ${OCCLUM_INSTANCE_DIR}/build/lib/libocclum-pal.so /usr/lib/liberpal-occlum.so
+COPY ${OCCLUM_INSTANCE_DIR}/build/lib/libocclum-pal.so.0.14.0 /usr/lib/libocclum-pal.so.0.14.0
```

Comment:
In addition, copy pal become unnecessary because pal is always loaded from host rather than container as before.